WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -285,5 +285,59 @@ public static async Task<ISet<uint256>> GetAllDependentsAsync(this IRPCClient rp
 
 			return hashSet;
 		}
+
+		public static async Task<VerboseBlockInfo> GetVerboseBlockAsync(this IRPCClient rpc, uint256 blockId)
+		{
+			var resp = await rpc.SendCommandAsync(RPCOperations.getblock, blockId, 3).ConfigureAwait(false);
+			var blockInfoStr = resp.Result.ToString();
+			var blockInfoJson = JObject.Parse(blockInfoStr);
+			var previousblockhash = blockInfoJson.Value<string>("previousblockhash");
+			var blockInfo = new VerboseBlockInfo()
+			{
+				Hash = uint256.Parse(blockInfoJson.Value<string>("hash")),
+				PrevBlockHash = previousblockhash is { } ? uint256.Parse(previousblockhash) : null,
```

Comment:
I don't like `null`s because you have to be checking nulls all the time.

```suggestion
				PrevBlockHash = previousblockhash is { } ? uint256.Parse(previousblockhash) : uint256.Zero,
```

Reply:
I did have zero there before but I think null is more correct because it represents mainly a genesis prevhash which is none existent.
Putting zero will actually simplify the coordinator code later, so I am indifferent.